GNUmakefile

```diff
@@ -31,6 +31,7 @@ gomodtidy: ## Run go mod tidy on all modules.
 	go mod tidy
 	cd ./core/scripts && go mod tidy
 	cd ./integration-tests && go mod tidy
+	cd ./integration-tests/load && go mod tidy
```

Comment:
Wen gomods?

Reply:
I don't know what that means... do you need me to do something here? I added this because it was missing.

Reply:
Nothing to do. This is necessary for now.
I was referring to https://github.com/jmank88/gomods